app/src/main/res/layout/activity_option1.xml

```diff
@@ -43,4 +43,28 @@
         android:textAppearance="@style/text_title"
         android:background="@color/colorAccent" />
 
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal">
+
+        <Button
+            android:id="@+id/back"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/touch_target_size"
+            android:layout_marginBottom="@dimen/small_spacing"
+            android:onClick="Back"
+            android:layout_weight="0.5"
+            android:text="back" />
+
+        <Button
+            android:id="@+id/next"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/touch_target_size"
+            android:layout_marginBottom="@dimen/small_spacing"
+            android:layout_weight="0.5"
+            android:onClick="Next"
```

Comment:
Suggestion: If you merge methods "Back" and "Next" into "Process", you can change the attribute to be "android:onClick="Process". So you will have only one method to handle all onClick events.